tests/native_tests/language.hy

```diff
@@ -860,6 +860,14 @@
   (assert (is (first []) None)))
 
 
+(defn test-slice []
+  (defclass CutTest []
+    (defn --getitem-- [self index]
+      (assert (= index (slice 1 6)))))
+
+  (get (CutTest) (: 1 6)))
+
+
 (defn test-cut []
```

Comment:
Shouldn't this be removed from `native_tests/langauge.hy` now that it's in `native_tests/native_macros.hy`?